### ISSUE TITLE
OBLIVION [BLUS30087] Autosave Issue

### ISSUE THREAD

**Issue by Darkhost1999** [RPCS3/rpcs3#12843]
### Quick summary

·F 3:42:28.054992 {PPU[0x100e004] Thread (thr_auto_save) [0x0016c994]} VM: Access violation reading location 0x0 (unmapped memory)

### Details

When playing Oblivion, particularly in Chorrol, and leaving from any of the inns, mostly The Grey Mare, a fatal error occurs **sometimes**. But always the autosave is fine and never gets corrupted.
If you play Oblivion for any duration you can see that the autosave message almost never leaves the screen because it frequently takes measurably 5 whole seconds or more to go away; particularly in Bravil which takes the longest time.
In reality, regardless of the message, any of the auto saves are loadable the moment you gain control over your character when walking through a door. The message lingering longer than necessary and this fatal error's behavior makes me assume something is taking WAY too long to exit the thr_auto_save thread because otherwise the game wouldn't actually save or the save would become corrupt. 
To reiterate the saves never do become corrupted or fail.

### Attach a log file

[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/9821114/RPCS3.log.gz)

### Attach capture files for visual issues

NA

### System configuration

R5 3600
GTX 1660S
WIN 11 Beta

### Other details

In summary there's 1 major issue
Autosave fatal error
and 1 minor issue
The message lingering longer than the PS3's. Even if the message has a minimum timeout, 5 whole seconds or longer isn't that minimum.

**Comment by d0x360** [comment]
Sound similar to a bug that game had on real PS3.  That's a good sign.

**Comment by Darkhost1999** [comment]
To copy from fandom

> Upon loading a save game, Oblivion runs fine. Freezing occurs three to four minutes into playing (ex; talking to townspeople, etc.). Another instance is a frozen black screen after entering or exiting buildings and fast traveling. During the freeze, on Xbox 360; the dashboard doesn't show up and the player cannot interact with the console at all, the only solution being to turn it off. Unconfirmed on PC and PS3 versions.

This would be great and all if it just froze. But since there's a fatal error message accompanied with this issue and the game already has notable saving issues when adjusting in game options. I don't think it's the same issue. I don't recall ever hearing of the issue before. Fandom saying only xbox 360 users is confirmed makes sense cause I've never seen or heard of the issue before. Not when watching others play or playing myself from original hardware.

To copy the solution

> Solution: Holding down the Y button on the loading screen will clear the loading cache, making the game playable again, but may delete the save.

In this case I assume they will mean to hold down triangle for PS3 but with loading screens being almost instant there really isn't an option for me to try that feature on RPCS3 the majority of the time.

**Comment by elad335** [comment]
Test with #13080

**Comment by Darkhost1999** [comment]
It's a rare bug triggered often by 1 bar in 1 town. I'm currently testing without trigger.

**Comment by Darkhost1999** [comment]
Indeed fixed. Just causes an extreme slow down now while game is saving instead of crashing. Behavior is expected from Bethesda and works as designed to my knowledge.
